Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:7 as node
+FROM eboraas/laravel
```

Comment:
It's not an official build from the `laravel` team.
Though it got quite a few stars and it's an automated build (source: https://bitbucket.org/EdBoraas/apache-docker/src/master/laravel/)